.github/workflows/pr-build-image.yaml

```
@@ -51,6 +49,11 @@ jobs:
 
       - name: Check if build should be skipped
         run: |
+          if [[ "${{ github.event.pull_request.base.ref }}" == "release-1.6" ]]; then
+            echo "::notice::Skipping build: PR targets release-1.6 branch"
+            exit 0
+          fi
+
           if [[ "${{ steps.check-image.outputs.relevant_changes }}" == "false" && "${{ steps.check-image.outputs.image_exists }}" == "true" ]]; then
```

this could also be enhanced to do "if commit message includes string "[skip-build]" then exit 0

```suggestion
          if [[ "${{ steps.check-image.outputs.relevant_changes }}" == "false" ]] && [[ "${{ steps.check-image.outputs.image_exists }}" == "true" ]]; then
```